experimental/dds/tree2/src/feature-libraries/forestSummarizer.ts

```diff
@@ -53,7 +69,12 @@ export class ForestSummarizer implements Summarizable {
 		this.forest.moveCursorToPath(undefined, this.cursor);
 		const fields = mapCursorFields(this.cursor, (cursor) => [
```

Comment:
One thing I'd like to fix (maybe as a separate change from yours) is that the summary is encoding each root field independently, which means each one gets its own copy of the schema information which is bad for compression.

```suggestion
		// TODO: Encode all detached fields in one operation for better performance and compression
		const fields = mapCursorFields(this.cursor, (cursor) => [
```

Reply:
added TODO comment.